### PR TITLE
Adding SVN-Partial Release Mgr plugin

### PR DESCRIPTION
**Pull request by IGiannis**
# Description

# Permissions pull request checklist

### Always

- [ ] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by daniel-beck** [comment]
> IGiannis wants to merge 1 commit into jenkins-infra:master from jenkinsci:master

What made you think forking into jenkinsci was the thing to do?

**Comment by daniel-beck** [comment]
> Add link to plugin/component Git repository in description above

https://github.com/jenkinsci/svn-partial-release-mgr-plugin

> Add link to resolved HOSTING issue in description above

https://issues.jenkins-ci.org/browse/HOSTING-296

**Comment by daniel-beck** [comment]
https://github.com/jenkinsci/svn-partial-release-mgr-plugin/blob/e4b79600c5ce4d06c1018f3057e92aaa2ab0a9a5/pom.xml#L5

This seems pretty weird for a new plugin. Not a blocker, but weird in any case.

File: permissions/plugin-svn-partial-release-mgr.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "svn-partial-release-mgr"
+paths:
+- "org/jenkins-ci/plugins/svn-partial-release-mgr"
```
Comment (daniel-beck) [inline permissions/plugin-svn-partial-release-mgr.yml]: This path is wrong. Please follow the instructions in this repository's readme, as also linked from the PR template. Both exist for a reason.

**Comment by IGiannis** [comment]
Hi daniel,
Ok I changed the group id into the pom (it was a little weird anf was forgotten there by mistake).
I guess the yml files does not need any changes now.

About the forking , I am new to git and github and it wasn't clear to me how to create the PR even after reading the readme.

BR,

**Comment by daniel-beck** [comment]
>  Ok I changed the group id into the pom (it was a little weird anf was forgotten there by mistake).

I think me cleaning up the wrong fork made this PR uneditable, so you need to file a new one from a fork in your personal account.

**Pull request closed without merging**